Scripts assume gnu versions of grep, awk, etc. so put /usr/gnu/bin first in
the $PATH on Solaris.

Not suitable for upstream.

--- xdg-utils-1.1.1/scripts/xdg-utils-common.in
+++ xdg-utils-1.1.1/scripts/xdg-utils-common.in
@@ -1,3 +1,4 @@
+export PATH="/usr/gnu/bin:/usr/bin:${PATH:-/usr/bin}"
 
 #----------------------------------------------------------------------------
 #   Common utility functions included in all XDG wrapper scripts
